### DISCUSSION TITLE
Cell size

### DISCUSSION THREAD

**Discussion by ahsadeghi** [pybamm-team/PyBaMM#3037] [Q&A]
Hi,
Is there any option in Pybamm to set the cell actual size/type (e.g., 18650, 21700, prismatic, etc.). If not, how is it managed?
Thank you.

Hosein

**Comment by valentinsulzer** [comment]
Most of the models available are 1D through the electrode stack so the form factor doesn't matter. The form factor only comes into these models via the parameters "Electrode height [m]" and "Electrode width [m]", which are multiplied to get the electrode area, then electrode area is used to convert from applied current in Amps to current density in A/m^2.

For prismatic/pouch cells, you can directly input those parameters from the cell dimensions. For jellyrolls, you can calculate them using [this formula](https://www.batterydesign.net/cylindrical-cell-electrode-estimation/). 

If you are interested in seeing effects beyond 1D, there is work from pybamm developers on current/temperature distributions in [jellyrolls](https://iopscience.iop.org/article/10.1149/1945-7111/aba44b/pdf) and [pouch/prismatic cells](https://arxiv.org/pdf/2008.03691.pdf).
